grafana/resource_api_key_test.go

```diff
@@ -62,9 +62,15 @@ func testAccGrafanaAuthKeyCheckDestroy(s *terraform.State) error {
 			return err
 		}
 
-		_, err = c.DeleteAPIKey(id)
-		if err == nil {
-			return fmt.Errorf("api key still exists")
+		keys, err := c.GetAPIKeys(false)
```

Comment:
The only code change I did is here. This check didn't work for v7.5. `DeleteAPIKey` worked on an already deleted key on that version. `GetAPIKeys` returns the correct information for every version